beets/mediafile.py

```diff
@@ -2005,6 +2005,15 @@ def update(self, dict):
         out_type=float,
     )
 
+    # EBU R128 fields.
+    r128_track_gain = MediaField(
+        StorageStyle(
+            u'R128_TRACK_GAIN',
+            float_places=0
```

Comment:
You don't need `float_places` if you don't intend to store floating-point numbers. Is this field supposed to store an integer?